packages/cypress/src/executors/cypress/cypress.impl.spec.ts

```diff
@@ -46,6 +46,9 @@ describe('Cypress builder', () => {
   (devkit as any).readTargetOptions = jest.fn().mockReturnValue({
     watch: true,
   });
+  (devkit as any).getExecutorInformation = jest
+    .fn()
+    .mockReturnValue({ schema: { properties: {} } });
```

Comment:
Does `jest.spyOn(devkit, 'getExecutorInformation').mockReturnValue(...)` not work here?

Reply:
changed